pkg/segment/aggregations/segaggs.go

```diff
@@ -1931,14 +1933,60 @@ func performMultiValueColRequestWithoutGroupby(letColReq *structs.LetColumnsRequ
 		case "makemv":
 			finalValue := performMakeMV(fieldValueStr, mvColReq)
 			rec[mvColReq.ColName] = finalValue
+		case "mvexpand":
+			expandedRecs := performMVExpand(fieldValue)
+
+			// Apply limit if mvColReq.Limit is greater than 0
+			if mvColReq.Limit > 0 && len(expandedRecs) > int(mvColReq.Limit) {
+				expandedRecs = expandedRecs[:mvColReq.Limit]
+			}
+			delete(recs, key)
+			for i, expandedValue := range expandedRecs {
+				expandedValueStr, ok := expandedValue.(string)
+				if !ok {
+					expandedValueStr = fmt.Sprintf("%v", expandedValue)
+				}
```

Comment:
Why are you converting this to string? You can just assign the value you are getting right? We might not want to convert Number types to strings.

Reply:
our representation of multi-value is always []string right? so even if we have numbers it always comes as string. But I can change it back to use `expandedValue` as it is so if we update the multi-value representation, then numbers or other data types don't get converted to strings.